src/Compilers/CSharp/Test/Semantic/FlowAnalysis/LocalFunctions.cs

```diff
@@ -699,21 +699,21 @@ void Local()
     }
 }");
             comp.VerifyDiagnostics(
-                // (19,9): error CS0165: Use of unassigned local variable 'a1'
-                //         Local1();
-                Diagnostic(ErrorCode.ERR_UseDefViolation, "Local1()").WithArguments("a1").WithLocation(19, 9),
-                // (28,9): error CS0170: Use of possibly unassigned field 'a'
+                // (19,9): error CS8079: Use of possibly unassigned auto-implemented property 'y'
```

Comment:
> // (19,9): error CS8079: Use of possibly unassigned auto-implemented property 'y' [](start = 15, length = 82)

It feels like the error should at least mention containing type for the property, it is hard to figure out how to address the error otherwise. Ideally we should give the whole chain to the top level local.